.github/ISSUE_TEMPLATE/performance_issue.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Performance issue
+about: Report a performance problem or regression
+title: ''
+labels: 'tenet-performance'
+assignees: ''
+
+---
+
+<!--This is just a template - feel free to delete any and all of it and replace as appropriate.-->
+
+#### Description
```

Comment:
Given that API proposals tend to be large documents, I would generally prefer it if headers were at least as large as [h2](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#h2). Most API proposals I sampled seem to be using h1.